release.mk

```diff
@@ -852,15 +845,9 @@ $O/.stamp_ide_linux_tar: \
 $O/.stamp_ide_macos_eclipse: $O/$(ECLIPSE_VANILLA_MACOS_TGZ) $O/$(IDE_PLUGINS_ZIP)
 	mkdir -m775 -p $O/$(IDE_MACOS_INSTALL)
 	tar xf $< -C $O/$(IDE_MACOS_INSTALL)
-	$O/$(IDE_MACOS_INSTALL)/Eclipse.app/Contents/MacOS/eclipse \
-            -application org.eclipse.equinox.p2.director \
-            -noSplash \
-            -repository $(ECLIPSE_REPO),jar:file:$(realpath $O/$(IDE_PLUGINS_ZIP))\!/ \
-            -installIU $(ECLIPSE_PREREQ),com.arc.cdt.feature.feature.group
-	# Eclipse will create a bunch of repos with local paths, that will not
-	# work for end-users, hence those repos must be manually removed.
-	sed -i -e "/$(subst /,_,$O)/ d" \
-            $O/$(IDE_MACOS_INSTALL)/Eclipse.app/Contents/Eclipse/p2/org.eclipse.equinox.p2.engine/profileRegistry/epp.package.cpp.profile/.data/.settings/org.eclipse.equinox.p2.*
+	unzip ${IDE_PLUGINS_ZIP} -d $O/$(IDE_LINUX_INSTALL)/eclipse/dropins
```

Comment:
Unzip path is not correct, it is `IDE_MACOS_INSTALL`. In addition you need to check if internal folder layout of Eclipse on macOS is same or different from Linux.